api/webhook/post.go

```diff
@@ -779,34 +779,16 @@ func handleRepositoryEvent(c *gin.Context, m *types.Metadata, h *library.Hook, r
 // renameRepository is a helper function that takes the old name of the repo,
 // queries the database for the repo that matches that name and org, and updates
 // that repo to its new name in order to preserve it. It also updates the secrets
-// associated with that repo.
+// associated with that repo as well as build links for the UI.
 func renameRepository(h *library.Hook, r *library.Repo, c *gin.Context, m *types.Metadata) (*library.Repo, error) {
 	logrus.Debugf("renaming repository from %s to %s", r.GetPreviousName(), r.GetName())
```

Comment:
what do you think about increasing log level here? feels like something that should be in audit and not only visibly in debug mode.